File: Rust/Easy/Accepted/detect-capital_2.rs
impl Solution {
    pub fn detect_capital_use(word: String) -> bool {
        
        let mut count = 0;
        for c in word.chars() {
            if c.is_uppercase() {
                count += 1;
            }
        }
        return count == word.len() || count == 0 || (count == 1 && word.chars().nth(0).unwrap().is_uppercase());
    }
}

File: Java/Easy/Accepted/detect-capital_2.java
class Solution {
    public boolean detectCapitalUse(String word) {
        
        int count = 0;
        for (char c : word.toCharArray()) {
            if ('Z' - c >= 0) {
                count++;
            }
        }
        return count == 0 || count == word.length() || (count == 1 && 'Z' - word.charAt(0) >= 0);
    }
}